src/java.base/share/classes/java/lang/ref/Finalizer.java

```diff
@@ -61,15 +61,13 @@ private Finalizer(Object finalizee) {
         return queue;
     }
 
-    static class Holder {
-        static final boolean ENABLED = isFinalizationEnabled();
-    }
+    static final boolean ENABLED = isFinalizationEnabled();
```

Comment:
private?

Reply:
Yeah, probably should be private. Other stuff in this class is private except things that are used from outside.